### ISSUE TITLE
Sysfs temperature configuration

### ISSUE THREAD

**Issue by Mihara** [liquidprompt/liquidprompt#807]
More configuration options for temperature reading, or an option to explicitly select the backend used.

### Description

When sensors report bogus temperatures for whatever reason, (which is what happened to one of my SSDs, which has a sensor with some kind of multiplier that the system doesn't know) I can add a configuration file to `lm-sensors` telling it to ignore that. I have no clue where would I do that with a sysfs temperature reading interface, or whether I even can.

I don't see a documented configuration option in `liquidprompt` letting me switch the backend  back to `sensors`, making temperature warning in the prompt useless, because one sensor always reports ~74 degrees, when every other sensor reports temperatures in the 30-40 degrees range.

The optimal solution would be a way to configure exceptions and ignore specific sensors, but the ability to manually select the backend used would also suffice.

### How will this be useful?

I will be able to turn LM_ENABLE_TEMP back on.

**Comment by Rycieos** [comment]
Thanks for the suggestion. I agree we need a way to silence/ignore sensors.

Your suggestion would be pretty simple to add. But there is another option. We could add a config array that would let you set devices to ignore in the sysfs backend. That would likely be faster than `sensors`, but it would be a more complicated feature. Thoughts?

**Comment by Mihara** [comment]
Would it help against a device that lm-sensors reports like this?

```
nvme-pci-0200
Adapter: PCI adapter
Composite:    +36.9°C  (low  = -20.1°C, high = +83.8°C)
                       (crit = +88.8°C)
Sensor 2:     +74.8°C
```

The `Composite` part, as far as I can tell, is correct, and I'd rather keep it. With lm-sensors, adding this to a file in `/etc/sensors.d` is sufficient to ignore this bogus reading and nothing else:

```
chip "nvme-pci-0200"
  ignore temp3
```

I am, unfortunately, not aware of how the sysfs backend works (finding out that temperatures were exposed over there was a surprise, even) so I don't know if I could disable individual sensors this way. If I could, then yes, a config array allowing me to disable individual bogus readings would be optimal, but disabling the entire device wholesale sounds like a bad idea.

**Comment by Rycieos** [comment]
The interface is very simple. We simply read all files that match these globs:
https://github.com/liquidprompt/liquidprompt/blob/e1a133bf92426ce493b56e967335059a56f34d00/liquidprompt#L615-L622

If you check on your system for files like those, you would likely see one that contains `83800` (units are millidegrees Celsius). That would be your bad sensor.

**Comment by Mihara** [comment]
> That would be your bad sensor.

The obvious solution would be to exclude specific ones ones by full path, then.

**Comment by Rycieos** [comment]
Yes, and that seems very doable. Which solution would you find more helpful?

**Comment by Mihara** [comment]
Excluding specific sysfs files by name (or better yet, a glob pattern?) would be just what I need.

**Comment by Rycieos** [comment]
Released in [v2.2.0-rc.2](https://github.com/liquidprompt/liquidprompt/releases/tag/v2.2.0-rc.2).
